pika/adapters/select_connection.py

```diff
@@ -367,11 +371,11 @@ def _process_fd_events(self, fd_event_map, write_only):
                 handler = self._fd_handlers[fileno]
                 handler(fileno, events, write_only=write_only)
 
-    def flush_outbound(self):
+    def _flush_outbound(self):
```

Comment:
No, that was existing code, the bug was that I'd accidentally renamed it in my original patch. It is removed by #556, but the other changes in there are required in order to do so and not create performance problems.